mautic/mautic
# #TPROD-309 - Migrate Category Bundle to Twig templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #TPROD-309 



#### Description:
Targeting the 5.x branch this PR will migrate Category bundle views from PHP templates to Twig templates


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. In the top right corner of the screen, open the "Settings" menu and click "Categories".
3. Test every single functionality on this page. Creating/editing/cloning webhooks, filtering, pagination, etc. - all functionality should work exactly like it did before.
